app/assets/javascripts/settings-controller.coffee

```diff
@@ -59,14 +60,40 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>
-
+  </div>                           
 </div>
+
+
+<h1 class="settings-subheader">Works in Progress</h1>
+<a><center> Here you can access all the models that have been edited</center></a>
+  
+<ul>
+  {{#each workInProgressLinks}}
+    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
+    <br>
+  {{/each}}
+</ul>
+
 """
+formatLinks = (wipStorage) ->
+  result = []
+
+  for url, model of wipStorage.inProgress
+    if model.title and model.timeStamp
+      formattedUrl = "/launch#http://" + url.replace(/^url:\/\//, '')
+      result.unshift({
+        modelTitle: model.title,
+        url: formattedUrl,
+        dataAccessed: new Date(model.timeStamp).toLocaleString()
```

Comment:
Seeing how the "Date Last Modified: " looks in the interface, I think maybe changing it to something like, "changed on" would be better.  Or maybe dropping it altogether, as it'll be clear from context what is being presented?  Also, it'd be great if the date string could be formatted like, **August 28th, 2023** just to make it a little more reader-friendly.